doc/setup.md

```diff
@@ -0,0 +1,286 @@
+
+```
+tapyrus-setup sign -static_vss=1:<static_vss[1, i]> -static_vss=2:<static_vss[2, i]> ... -static_vss=t:<static_vss[t, i]> -ephemeral_vss=1:<ephemeral_vss[1, i]> -ephemeral_vss=2:<ephemeral_vss[2, i]> ... -ephemeral_vss=t:<ephemeral_vss[t, i]>   -aggregated_public_key=<aggregated_public_key> -nodesecret=<node_secret[i]> -private_key=<static_private_key[i]> -block=<block>
+
+output: <local_sig[i]>
+```
+
+- `static_vss[j, i]` (j = 1, 2, ..., t) are the static VSSs generated by Step 3.
+- `ephemeral_vss[j, i]` (j = 1, 2, ..., t) are the ephemeral VSSs generated by Step 7.
+- `static_private_key[i]` is the static private key of Signer[i], generated in Step 2.
+- `aggregated_public_key` is an aggregated public key, generated in Step 4.
+- `node_secret[i]` is the secret key share of Signer[i], generated in Step 4.
+- `block` is the genesis block without block proof.
+- `local_sig[i]` is the "local signature" described as γi in the paper [^4]. signer[i] may reveal `local_sig[i]`.
+
+Then each signer broadcasts their local_sig.
+
+## Step 9. Compute the signature of the genesis block.
+
+After collecting `local_sig`, we can compute the signature of the genesis block using `tapyrus-setup combinesig`
+
+```
+tapyrus-setup combinesig -sig=<local_sig[1]> -sig=<local_sig[2]> ... -sig=<local_sig[t]> -block=<block>
+
+output: <block_with_signature>
+```
+
+- `local_sig[i]` is the local signatures broadcasted by signers.
+- `block_with_signature` is the whole genesis block data with block proof as hex string format.
+
+To make block proof, t `local_sig`s are required.
+`local_sig`s generated by signers are public values, so signers can reveal them as plaintext without any encryption.
+
+## Step 10. genesis.`networkid`
+
+Create file named 'genesis.`networkid`' and fill it with `block_with_signature`.
+
+## Next Step
+
+Now we are ready to start tapyrus core and tapyrus signer network.
+
+To start tapyrus core network, see [Start Tapyrus-core nodes](https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/getting_started.md#5-start-tapyrus-core-nodes).
+
+To start tapyrus signer network, go to [How To configure Tapyrus Signer Network](./configuration.md). The secret values or keys refered in this document will be required to start Tapyrus Signer Network.
+
+- `static_private_key[i]` and `static_public_key[i]` in Step 1
+- `aggregated_public_key` and `node_secret[i]` in Step 3
+
+In above list, `static_private_key` and `node_secret` must be treated as secret.
+
+## Appendix A: Encoding and Encrypting the VSS
+
+In this section, we describe the protocol to encode/encrypt the VSS.
+`static_vss[i, j]` in Step 3 and `ephemeral_vss[i, j]` in Step 7 have same structure.
+
+Below in this section, notation `vss` means `static_vss[i, j]` or `ephemeral_vss[i, j]`.
+
+### Structure of static vss
```

Comment:
We need to show encodings for other data like `local_sig`, `block`.

I think below is ok

* block: The structure of block is (here)[https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/signedblocks.md#block-structure-expansion-for-signed-blocks]. And it is encoded as hex string combined all fields.
* local_sig: local_sig is encoded hex string of ( r | s ).
     * r - 32 bytes. the unsigned big-endian 256-bit encoding of the Schnorr signature's r integer.
     * s - 32 bytes. the unsigned big-endian 256-bit encoding of the Schnorr signature's s integer.

Reply:
Fixed at 18f02b7605865f3b7fb39293f3f051a9958dfcb9